narayana/tools/src/main/java/io/narayana/perf/jmh/Benchmark.java

```diff
@@ -71,15 +71,40 @@ public void setPrevious(Benchmark previous) {
     @Override
     public String toString() {
         if (previous != null) {
-            double pc = ( score / previous.score - 1) * 100.0;
-            String res = regression == -1 ? "regression" : regression == 0 ? "no change" : "improvement";
-            return String.format("%s: %f vrs %f (%f%%: %s)%n",
-                    benchmark, score, previous.score, pc, res);
+            double percentChange = 0.0;
+            boolean isIntersecting = scoresIntersecting();
```

Comment:
This new code is testing two different benchmarks so it would be preferable if the logic was moved into the comparison method:

``` java
public int compareTo(Benchmark other) {
   // the code that compares two benchmarks taking account of the score error
```

Then the toString() will not need to change.

It would also mean that the 

```java
public void setPrevious(Benchmark previous) {
        this.previous = previous;

        regression = compareTo(previous);
    }
```

would take account of the error.

Reply:
Okay, I will refactor it.